selfservice/flow/login/handler_test.go

```diff
@@ -553,18 +554,24 @@ func TestGetFlow(t *testing.T) {
 	})
 
 	t.Run("case=expired with return_to", func(t *testing.T) {
```

Comment:
I ran these tests without your modification and they passed, can you please revert this test and add tests that would fail without the modifications? Preferrably:

1. tests that reside in `/selfservice/flow/<the-flow-type>/test` where you create a flow and then get it, and expect the return url to be set.
2. At least one test that would otherwise fail, either in handler_test or in the e2e tests

Reply:
Nice work :D I haven't used Soda before but I was going to suggest using post-load hooks if they were available so great to see it working!

Hmm... on reverting back to my commit, commenting out the r.SetReturnTo in persister_login.go, I get the following:

<img width="508" alt="Screen Shot 2022-03-28 at 5 45 01 pm" src="https://user-images.githubusercontent.com/16401696/160350673-7196c5e5-dfd2-40da-9fa1-036c51c8bb9d.png">

I am running the tests with `go test -p 1 -tags sqlite -count=1 -failfast ./selfservice/flow/login`

Reply:
You're right, not sure why it passed for me! Thank you for validating :)